docs/sources/features/datasources/azuremonitor.md

```diff
@@ -31,7 +31,7 @@ The data source can access metrics from four different services. You can configu
 - [Guide to setting up an Azure Active Directory Application for Azure Log Analytics.](https://dev.loganalytics.io/documentation/Authorization/AAD-Setup)
 - [Quickstart Guide for Application Insights.](https://dev.applicationinsights.io/quickstart/)
 
-1. Accessed from the Grafana main menu, newly installed data sources can be added immediately within the Data Sources section. Next, click the "Add data source" button in the upper right. The Azure Monitor data source will be available for selection in the Cloud section in the list of data sources.
+1. Accessed from the Grafana main menu, newly installed data sources can be added immediately within the data sources section. Next, click the "Add data source" button in the upper right. The Azure Monitor data source will be available for selection in the Cloud section in the list of data sources.
```

Comment:
This refers to the section in the main menu, where it says "Data Sources", so we should leave this one as is.